### ISSUE TITLE
Fails to parse relative URIs

### ISSUE THREAD

**Issue by tpluscode** [rdfjs/rdfxml-streaming-parser.js#41]
I think related to #20 but maybe not exactly the same, I found out that the parser fails to handle some relative named nodes and throws. For example I failed to parse `https://www.w3.org/ns/earl#` and it fails right off the bat on `<owl:Ontology rdf:about="">`

I am running this from `@rdfjs/fetch` which sets a `baseIRI` parameter when calling like `parser.import({ baseIRI: res.url })`.

The parser however does not take a second parameter and the base test ignores. I think it would be worth adding, similarly to how most other RDF/JS parser out there do

**Comment by rubensworks** [comment]
Seems like the same problem as in #32. But this should've been fixed in 1.3.5. Are you sure you're using the latest version?

**Comment by tpluscode** [comment]
In fact I was not and yes, the parsing is fixed when a baseIRI is passed to the constructor. Would still be nice to add a parameter to `import` though

**Comment by rubensworks** [comment]
Makes sense, but we may want to define this as spec-level first, as it does not seem to allow parameters to be passed: http://rdf.js.org/stream-spec/#sink-interface

**Comment by tpluscode** [comment]
Not a bad idea, at least for `baseIRI` to be standardised like the ctor parameter.

On the other hand, since we're talking about JS, any superfluous parameters are also not forbidden implicitly. 

Are you going to create an issue?

**Comment by rubensworks** [comment]
> Are you going to create an issue?

I don't have the available bandwidth to pull this myself, so feel free to create an issue! :-)
